coverage/.resultset.json

```diff
@@ -524,5 +524,531 @@
       ]
     },
     "timestamp": 1521158702
```

Comment:
the whole coverage folder should be gitignored. We should have done this already but would you mind